doc/source/appendix/ontologies.rst

```diff
@@ -84,122 +89,142 @@ Examples
 Genotypic sex
 =============
 
-:ontologyId:
-	"http://purl.obolibrary.org/obo/PATO_0020001",
+:termId:
+  "PATO:0020001",
 :term:
   "male genotypic sex" ,
+:termUri:
+  "http://purl.obolibrary.org/obo/PATO_0020001"
 :sourceName:
-	"PATO Phenotypic quality",
+  "PATO Phenotypic quality",
 
 
 Sequence Ontology
 =================
 
-:ontologyId:
-  "http://purl.obolibrary.org/obo/SO_0001583",
+:termId:
+  "SO:0001583",
 :term:
-	"missense_variant",
+  "missense_variant",
+:termUri:
+  "http://purl.obolibrary.org/obo/SO_0001583",
 :sourceName:
-	"Sequence Ontology",
+  "Sequence Ontology",
 :sourceVersion:
   "release_2.5.3"
 
 
 Human Phenotype ontology
 ========================
 
-:ontologyId:
-  "http://purl.obolibrary.org/obo/HP_0000819",
+:termId:
+  "HP:0000819",
 :term:
-	"Diabetes mellitus",
+  "Diabetes mellitus",
+:termUri:
+  "http://purl.obolibrary.org/obo/HP_0000819",
 :sourceName:
-	"human_phenotype Ontology",
+  "human_phenotype Ontology",
 :sourceVersion:
-	"release_Jan2016*"
+  "release_Jan2016*"
```

Comment:
in the obo-file there is a "data-version" tag. I don't know right now how the annotation property is called in owl.

Sent from mobile

> On 08.09.2016, at 20:02, Julie McMurry <notifications@github.com> wrote:
> 
> In doc/source/appendix/ontologies.rst:
> 
> >  :sourceVersion:
> > -	"release_Jan2016*"
> > +  "release_Jan2016*"
> @drseb @cmungall what is the way that HP versions should be denoted?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>